test/fixture/terraform_12/aws_tags_merge/expected/main.terratag.tf

```diff
@@ -0,0 +1,32 @@
+provider "aws" {
+  version = "~> 2.0"
+  region  = "us-east-1"
+}
+
+variable "name" {
+  type    = string
+  default = ""
+}
+
+variable "tags" {
+  type    = map(string)
+  default = {}
+}
+
+resource "aws_s3_bucket" "example" {
```

Comment:
@avnerenv0 is there any special reason why `tf12` scenario/tests are different than `tf11`? 🤔

Reply:
just because the interpolation syntax `"${...}"` I wanted to make sure it works as expected